package.json

```diff
@@ -117,6 +117,7 @@
 				"rules": {
 					"@typescript-eslint/no-misused-promises": "off",
 					"@typescript-eslint/promise-function-async": "off",
+					"@typescript-eslint/no-explicit-any": "error",
```

Comment:
- It was disabled upstream https://github.com/xojs/eslint-config-xo-typescript/commit/bbbf88972520e8af3439ba2ab3f1a1c40e68041c
- But here its support was fixed: https://github.com/sindresorhus/refined-github/pull/2429